recipes/maxentscan/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: maxentscan
+  version: "1.0"
+source:
+  fn: fordownload.tar.gz
+  url: http://genes.mit.edu/burgelab/maxent/download/fordownload.tar.gz
+  md5: 3d11d5ca02911e6b87eb6fdca38ab631
+
+build:
+    number: 0
+    skip: True # [osx]
+requirements:
+  build:
+    - perl-threaded
+  run:
+    - perl-threaded
+test:
+  commands:
+    - perl maxentscan_score3.pl
```

Comment:
The tests should work without adding perl. If not, the shebang line of the scripts has to be fixed.